test/messages/chainlock.js

```diff
@@ -0,0 +1,174 @@
+'use strict';
+
+var chai = require('chai');
+var should = chai.should();
+var expect = chai.expect;
+
+var bitcore = require('../../index');
+var ChainLock = bitcore.messages.ChainLock;
+var QuorumEntry = bitcore.QuorumEntry;
+
+describe('ChainLock', function () {
+  // Output from https://github.com/dashpay/dash/pull/3718 PR's description
+  var object = {
```

Comment:
Could you move that to `beforeEach` clause, so it instantiates before each test and won't be accidentally modified? Thank you!

Reply:
Done !